src/api/app/controllers/webui/packages/bs_requests_controller.rb

```diff
@@ -12,7 +12,7 @@ def index
         if Flipper.enabled?(:request_index, User.session)
           # FIXME: Once we roll out filter_requests should become a before_action
           filter_requests
-          @bs_requests = @bs_requests.page(params[:page])
+          @bs_requests = @bs_requests.order('number DESC').page(params[:page])
```

Comment:
Any specific reason why we order by `number` by default instead of `created at`? I know the `number` is sequential, but it doesn't really reflect the logic, and who knows if `number` sticks to the rule always forever.

Reply:
No just because we did this before. It's the same...